README.md

```diff
@@ -46,7 +46,7 @@ yarn start # start local dev server
 
 ```sh
 rustc --version
-# rustc 1.65.0 (897e37553 2022-11-02)
+# rustc 1.69.0 (84c898d65 2023-04-16)
```

Comment:
I see that the Solana crates bumped the MSRV to `1.68.0`, let's update this to latest stable `rustc 1.71.0 (8ede3aae2 2023-07-12)` while we are at it.

Reply:
In the Solana repo they use the 1.69 rust version but yeah anyways let's use the latest